README.md

```diff
@@ -1,218 +1,207 @@
 
 ```javascript
-LookerEmbedSDK.init('looker.example.com', '/auth')
+getEmbedSDK().init('looker.example.com', '/auth')
 ```
 
-In this example, `/auth` is a backend service that you must implement as described in the [Auth](#the-auth-endpoint) section.
+In this example, `/auth` is a backend service that must be implemented as described in the [Signed URL Auth Endpoint](#the-auth-endpoint) section.
 
-After the SDK is initialized, begin by creating the builder with an `id`. For example, to create a dashboard embed builder:
+After the SDK is initialized, create the builder using an `id` or `URL`. For example, to create a Dashboard embed builder do one of the following:
 
 ```javascript
-LookerEmbedSDK.createDashboardWithId(id)
+getEmbedSDK().createDashboardWithId('42')
+// OR
+getEmbedSDK().createDashboardWithUrl('/embed/dashboards/42?state=california')
 ```
 
-You can then add additional attributes to the builder to complete your setup:
+The `create[ContentType]WithUrl` functions can used with both signed and cookieless embed starting with Embed SDK 2.0.0. Prior to Embed SDK 2.0.0, Cookieless Embed only supported `create[CotentType]WithId`.
+
+Additional attributes can be added to the builder to complete the setup:
 
 ```javascript
   .appendTo('#dashboard')
 ```
 
-You can add event handlers:
+Event handlers can be added:
 
 ```javascript
   .on('dashboard:run:start',
-      () => updateState('#dashboard-state', 'Running')
+      () => updateStatus('Running')
   )
   .on('dashboard:run:complete',
-      () => updateState('#dashboard-state', 'Done')
+      () => updateStatus('Done')
   )
 ```
 
-You finish by building the embedded element:
+Create an embedded client by calling the build method:
 
 ```javascript
   .build()
 ```
 
-The `createDashboardWithId` function will call your backend `/auth` endpoint when `build` is invoked and requires a signed embed URL in response. Subsequent embeds can be generated using `createDashboardWithUrl` which accepts a partial URL matching [this form](https://docs.looker.com/reference/embedding/sso-embed#building_the_embed_url), for example: `/embed/dashboards/`. The URL create functions will not call your backend `/auth` service. If you are embedding multiple items on a single page, use ID create functions first and then URL create functions subsequently to avoid redundant calls to your auth backend.
+### Connecting
 
-If you want to send and receive messages to the embedded element you need to call `connect()` which returns a Promise that resolves to the communication interface of the given element:
+Once the client is built, call connect to create the IFRAME. The connect process creates the `src` attribute used for the actual IFRAME. How the `src` value is generated is based upon how the embed SDK is initialized.
```

Comment:
Since I'm suggesting another edit:
```suggestion
Once the client is built, call `connect` to create the IFRAME. The connect process creates the `src` attribute used for the actual IFRAME. How the `src` value is generated is based upon how the embed SDK is initialized.
```